packages/expo-build-properties/src/ios.ts

```diff
@@ -49,6 +49,10 @@ export const withIosBuildProperties = createBuildPodfilePropsConfigPlugin<Plugin
       propValueGetter: (config) =>
         (config.ios?.privacyManifestAggregationEnabled ?? true).toString(),
     },
+    {
+      propName: 'ios.buildFromSource',
+      propValueGetter: (config) => (config.ios?.buildFromSource ?? true)?.toString(),
```

Comment:
```suggestion
      propValueGetter: (config) => config.ios?.buildFromSource?.toString(),
```

that would prevent `"ios.buildFromSource": "false"` pulled into Podfile.properties.json even user does not define the `ios.buildFromSource in their build-properties config.

you have the `if podfile_properties['ios.buildFromSource'] == 'false'` in podfile. even the `ios.buildFromSource` is not defined in Podfile.properties.json. it should still build from source. which is great 👍